DSharpPlus/Entities/Channel/Message/DiscordMessage.cs

```diff
@@ -317,6 +318,18 @@ public IReadOnlyList<DiscordMessageSticker> Stickers
         [JsonIgnore]
         private readonly Lazy<IReadOnlyList<DiscordMessageSticker>> _stickersLazy;
 
+        /// <summary>
+        /// Gets emojis for this message
+        /// </summary>
+        [JsonIgnore]
+        public IReadOnlyList<DiscordEmoji> Emojis
+            => this._emojisLazy.Value;
+
+        [JsonIgnore]
+        internal List<DiscordEmoji> _emojis = new();
+        [JsonIgnore]
+        private readonly Lazy<IReadOnlyList<DiscordEmoji>> _emojisLazy;
+
```

Comment:
Absolutely not. We no longer populate from content. If discord doesn't send it, we don't populate it.

Reply:
I'm a bit confused... Mentions (role mentions and channel mentions) are still populated from content (ref. `DiscordMessage.PopulateMentions`), that's why I thought that it is a good opportunity.
I probably could do it all locally in the method, that's not really a problem, but I really need some kind of a definitive decision on this point.

Reply:
@Emzi0767, this isn't technically accurate. In order to continue maintaining backwards compatibility, we do still populate the mentions from the content of the message. Or at least that was my thought process when I left it in at all. I can remove it though if we'd like to do that.

Reply:
Removal of such behaviour would definitely happen